src/ios/PushPlugin.m

```diff
@@ -495,6 +495,11 @@ - (void)clearAllNotifications:(CDVInvokedUrlCommand *)command
     [self.commandDelegate sendPluginResult:commandResult callbackId:command.callbackId];
 }
 
+- (void)listenForRegistration:(CDVInvokedUrlCommand *)command
+{
+    self.callbackId = command.callbackId;
```

Comment:
This instance variable is also set in `init`, and is called from `didRegisterForRemoteNotificationsWithDeviceToken`.

This change lets you register a callback without calling `init`